RecoMuon/MuonIdentification/python/muons1stStep_cfi.py

```diff
@@ -77,7 +77,10 @@
                                           OverlapDTheta = cms.double(0.02), # 1.14 degrees
                                           ClusterDPhi   = cms.double(0.6), # 34 degrees
                                           ClusterDTheta = cms.double(0.02) # 1.14
-    )
+    ),
+
+    # tracker muon arbitration
+    arbitrateTrackerMuons = cms.bool(True)
```

Comment:
The option is there. If we ever find a case where it needs to be set false it's easy to do. I really doubt that it will ever happen.